src/test/components/__snapshots__/FilterNavigatorBar.test.js.snap

```diff
@@ -67,22 +67,22 @@ exports[`app/ProfileFilterNavigator renders ProfileFilterNavigator properly 3`]
   <li
     class="filterNavigatorBarItem filterNavigatorBarSelectedItem filterNavigatorBarLeafItem filterNavigatorBarTransition-enter filterNavigatorBarTransition-enter-active"
     data-index="1"
-    title="40 ms"
+    title="40ms"
   >
     <span
       class="filterNavigatorBarItemContent"
     >
-      40 ms
+      40ms
     </span>
   </li>
   <li
     class="filterNavigatorBarItem filterNavigatorBarLeafItem filterNavigatorBarUncommittedItem filterNavigatorBarUncommittedTransition-enter filterNavigatorBarUncommittedTransition-enter-active"
-    title="10 ms"
+    title="100.0μs"
```

Comment:
Why is this `.0` at at the end? I would prefer it to show `100μs`.

Reply:
I think this is because I used "`formatTimestmap` with "1" as the `fractionalDigits`. Maybe I can use `0` and see what it gives.

TBH I wasn't so concerned because we don't usually have a range that's round :)

Reply:
So I tried various things, and I think I can't make it disappear. What happens here is that the value isn't `100.0` but `100.0XXXXX`. This will disappear if we move to integers ;)

In the mean time I changed the options for `significantdigits` and `maxFractionalDigits` to smaller values and I think it's nicer. We don't need too much precision for big values.

Reply:
Funnily, the `.0` was removed after rebasing on top of the URL range PR. I'm not sure what changed between the version that was already present in this PR and the version that landed...